### DISCUSSION TITLE
Postgres backups, branches and time travel

### DISCUSSION THREAD

**Discussion by MickL** [vercel/community#6034] [Feature Question]
### Question

Vercel provides Postgres databases but no possibility of making backups or having automatic backups. Is this feature planend? And if yes, when is it coming? Or do you really want all your customers to do backups manually with `pg_dump`?

Looking at Cloudflare they dont just provide backups but even the possbility to "time travel" to any minute within the last 30 days. Is this planned for Vercel Postgres as well?

Looking at Neon (which powers Vercel Postgres as far as I know) they provide database branching which seems VERY helpful for feature development. Is this planned for Vercel as well?

**Comment by jsgv** [comment]
I am also very interested in Neon's database branching. According to Vercel [documentation](https://vercel.com/docs/storage/vercel-postgres/limits#unsupported-neon-features) it will be supported in the future.

Is this weeks/months/years away?

**Comment by pawlean** [comment]
Hi @MickL and @jsgv!

Currently, automated backups are not supported [[1](https://vercel.com/docs/storage/vercel-postgres/faq#automated-backup-support)] however this is on our roadmap. 

We'll update this thread and wider community with any updates. Thanks for sharing your thoughts!

**Reply by jsgv** [reply]
Hi @pawlean , 

Thanks for the reply. My comment was actually regarding database branching.

[a comment by an automated account is not shown]
